docs/API.md

```diff
@@ -235,6 +235,18 @@ Returns:
 
 The storedId can be used in the api call /api/shortlinkinfo/<id> and to open in the website with a /z/<id> shortlink.
 
+### `GET /z/<id>` - Opens the website from a shortlink
```

Comment:
In future I was thinking we might want to namespace `<id>`s like `GET /z/<username>/tag`. I'm pretty sure we can get this to work in either case (and/or change that idea...which is a long way off anyway! Was just thinking out loud)